src/EPR.Payment.Service.Common.Data/AppDbContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using EPR.Payment.Service.Common.Data.DataModels.Lookups;
+using EPR.Payment.Service.Common.Data.Interfaces;
+using EPR.Payment.Service.Common.Data.SeedData;
+using Microsoft.EntityFrameworkCore;
+using System.Diagnostics.CodeAnalysis;
+
+namespace EPR.Payment.Service.Common.Data
+{
+    [ExcludeFromCodeCoverage]
+    public class AppDbContext : DbContext, IAppDbContext
```

Comment:
Replace the file with the one I had sent. Removed unused constructors

Reply:
resolved